### ISSUE TITLE
Users don't add all of their locations, like *ever*

### ISSUE THREAD

**Issue by jmadans** [codeforamerica/cityvoice-survey-builder#99]
In 3 out of 5 of our pilot, our city partners have not added all survey sites on the locations page of the builder. Then when they plead to add them after the fact we have to drop the database, edit the locations file, and then re import them. 

How do we mitigate the pain of this problem?

**Comment by daguar** [comment]
This is definitely an interesting observed user behavior. At a high level I think if the survey creator is already being assisted by someone like @jmadans, then it's this is all good — they built-app is in a "test" state that they're able to play around with and tweak with the help of the assister (changing the locations is not a time-intensive task.)

If we wanted CityVoice to be fully standalone usable by survey creators WITHOUT an assister, I think the best bet would be creation of an admin panel inside the application itself which allows the survey owner to log in and edit/add/delete locations.

(This relates to my long-term view that ultimately the best approach is to get ALL of the custom survey data into the database and out of version control; that's a cleaner separation of things but is a big chunk of work.)

**Comment by waltz** [comment]
Yeah, this is a super interesting phenomena. It definitely makes sense though. I forget things all the time, and have to go back and fix them.

I agree with @daguar that an administration panel would be the way forward. That project would have to be carefully navigated. I think that big extensions like are risky in that they can eat up a tremendous amount of time very easily.

**Comment by daguar** [comment]
Yeah, if the vision moving forward is that people are always assisted anyway, I see this as a thing the person assisting the govt partners can take care of with minimal effort.
